lib/response.js

```diff
@@ -1052,6 +1052,32 @@ function sendfile(res, file, options, callback) {
 }
 
 /**
+ * Run every tracers set at the application level for given events
+ * and arguments. To set a tracer on the application use the `instrument`
+ * method.
+ *
+ * @param {function} event The event to trace.
+ * @return {ServerResponse} for chaining
+ * @public
+ */
+
+res.trace = function(event) {
+  var args = [];
+  switch (arguments.length) {
+    case 0:
+      throw new Error('No event defined!');
+    case 1: break;
+    case 2:
+      args = [arguments[1]];
+      break;
+    default:
+      args = [].slice.call(arguments, 1);
+  }
+  this.app._callTracers(this, event, args);
```

Comment:
Hey @dougwilson I am not quite getting what you mean by 
>run under a different stack
Is there a snippet you can point to?
If you don't think it will impact the recording of the timestamp but will improve performance then as an end user it makes sense.